app/gui/src/model/suggestion_database.rs

```diff
@@ -915,6 +916,7 @@ mod test {
             name:                   "TextType".to_string(),
             module:                 "TestProject.TestModule".to_string(),
             params:                 vec![],
+            parent_type:            Some("Any".to_string()),
```

Comment:
This literal appears multiple times in the source code. This should never happen. All literals should be extracted to constants.